src/meltano/cli/__init__.py

```diff
@@ -97,6 +100,8 @@ def _run_cli():
     Raises:
         KeyboardInterrupt: if caught.
     """
+    logger.debug(f"meltano, version : {__version__}")
+    logger.debug(f"operating system : {os.name}")
```

Comment:
printf-style is preferred for logs, and we can probably emit a single event:

```python
logger.debug("meltano %s, %s", __version__, os.name)
```

https://blog.pilosus.org/posts/2020/01/24/python-f-strings-in-logging/

Reply:
done